src/wp-admin/includes/meta-boxes.php

```diff
@@ -193,56 +193,74 @@ function post_submit_meta_box( $post, $args = array() ) {
+		$date = sprintf(
+			$date_string,
+			date_i18n( $date_format, strtotime( $post->post_date ) ),
+			date_i18n( $time_format, strtotime( $post->post_date ) )
+		);
 	} else { // draft (no saves, and thus no date specified)
 		$stamp = __( 'Publish <b>immediately</b>' );
-		$date  = date_i18n( $datef, strtotime( current_time( 'mysql' ) ) );
+		$date  = sprintf(
+			$date_string,
+			date_i18n( $date_format, strtotime( current_time( 'mysql' ) ) ),
+			date_i18n( $time_format, strtotime( current_time( 'mysql' ) ) )
+		);
 	}
 
 	if ( ! empty( $args['args']['revisions_count'] ) ) :
 		?>
-<div class="misc-pub-section misc-pub-revisions">
-		<?php
-		/* translators: Post revisions heading. 1: The number of available revisions */
-		printf( __( 'Revisions: %s' ), '<b>' . number_format_i18n( $args['args']['revisions_count'] ) . '</b>' );
-		?>
-	<a class="hide-if-no-js" href="<?php echo esc_url( get_edit_post_link( $args['args']['revision_id'] ) ); ?>"><span aria-hidden="true"><?php _ex( 'Browse', 'revisions' ); ?></span> <span class="screen-reader-text"><?php _e( 'Browse revisions' ); ?></span></a>
-</div>
+		<div class="misc-pub-section misc-pub-revisions">
+			<?php
+			/* translators: Post revisions heading. 1: The number of available revisions */
+			printf( __( 'Revisions: %s' ), '<b>' . number_format_i18n( $args['args']['revisions_count'] ) . '</b>' );
+			?>
+		<a class="hide-if-no-js" href="<?php echo esc_url( get_edit_post_link( $args['args']['revision_id'] ) ); ?>"><span aria-hidden="true"><?php _ex( 'Browse', 'revisions' ); ?></span> <span class="screen-reader-text"><?php _e( 'Browse revisions' ); ?></span></a>
+		</div>
 		<?php
 	endif;
 
 	if ( $can_publish ) : // Contributors don't get to choose the date of publish
 		?>
-<div class="misc-pub-section curtime misc-pub-curtime">
-	<span id="timestamp">
-		<?php printf( $stamp, $date ); ?></span>
-	<a href="#edit_timestamp" class="edit-timestamp hide-if-no-js" role="button"><span aria-hidden="true"><?php _e( 'Edit' ); ?></span> <span class="screen-reader-text"><?php _e( 'Edit date and time' ); ?></span></a>
-	<fieldset id="timestampdiv" class="hide-if-js">
-	<legend class="screen-reader-text"><?php _e( 'Date and time' ); ?></legend>
-		<?php touch_time( ( $action === 'edit' ), 1 ); ?>
-	</fieldset>
-</div><?php // /misc-pub-section ?>
-	<?php endif; ?>
+		<div class="misc-pub-section curtime misc-pub-curtime">
+		<span id="timestamp">
+		<?php printf( $stamp, '<b>' . $date . '</b>' ); ?>
+		</span>
+		<a href="#edit_timestamp" class="edit-timestamp hide-if-no-js" role="button">
+			<span aria-hidden="true"><?php _e( 'Edit' ); ?></span>
+			<span class="screen-reader-text"><?php _e( 'Edit date and time' ); ?></span>
+		</a>
+		<fieldset id="timestampdiv" class="hide-if-js">
+			<legend class="screen-reader-text"><?php _e( 'Date and time' ); ?></legend>
+			<?php touch_time( ( $action === 'edit' ), 1 ); ?>
+		</fieldset>
+		</div>
+		<?php // /misc-pub-section ?>
```

Comment:
The comment and the closing php tag seems to lead to this:
<img width="289" alt="image" src="https://user-images.githubusercontent.com/29772709/190426193-d57f73ae-0a7c-4e14-b074-31d1e280d383.png">